web/controllers/auth_controller.ex

```diff
@@ -1,13 +1,26 @@
 defmodule RemoteRetro.AuthController do
   use RemoteRetro.Web, :controller
-  alias RemoteRetro.Google
+  alias RemoteRetro.OAuth.Google
+  alias RemoteRetro.User
 
   def index(conn, _params) do
     redirect conn, external: authorize_url!()
   end
 
   def callback(conn, %{"code" => code}) do
-    conn = put_session(conn, :current_user, Google.get_user_info!(code))
+    user_info = Google.get_user_info!(code)
+    conn = put_session(conn, :current_user, user_info)
```

Comment:
this one is a bit weird for me given that we're saying the session's `current_user` is really just the map of `google_user_info`, when in actuality our user model is a map that happens to have an attribute called `google_user_info` that has the same fields. Makes me feel more strongly that the `users` table should just have a column for each attribute.